src/main/scala/sigmastate/serialization/transformers/ByIndexSerializer.scala

```diff
@@ -15,12 +15,13 @@ object ByIndexSerializer extends ValueSerializer[ByIndex[SType]] {
   override val opCode: OpCode = OpCodes.ByIndexCode
 
   override def parseBody(bytes: Array[Byte], pos: Position): (ByIndex[SType], Consumed) = {
-    val (input, consumed) = ValueSerializer.deserialize(bytes, pos)
-    val index = Ints.fromByteArray(bytes.slice(pos + consumed, pos + consumed + intLength))
```

Comment:
Don't forget to remove unused imports like `com.google.common.primitives.Ints`

Reply:
@catena2w thanks! Fixed.